components/cop_datatype/src/def/eval_type.rs

```diff
@@ -18,6 +18,24 @@ pub enum EvalType {
     Json,
 }
 
+impl EvalType {
+    /// Converts `EvalType` into one of the compatible `FieldTypeTp`s.
+    ///
+    /// This function should be only useful in test scenarios that only cares about `EvalType` but
+    /// accepts a `FieldTypeTp`.
+    pub fn into_certain_field_type_tp_for_test(self) -> crate::FieldTypeTp {
```

Comment:
Does `#[cfg(test)]` need to be added?

Reply:
No, some compiler bugs will make it fail (cannot find this function).

Reply:
I think we should move this function into mod `test`

Reply:
Agree. But maybe not helpful to resolve the bug I mentioned above